test/collection_test.js

```diff
@@ -2200,14 +2185,14 @@ describe("Collection", () => {
       it("should transfer the headers option", () => {
         return articles.sync({headers: {Foo: "Bar"}})
           .then(() => {
-            expect(pullChanges.firstCall.args[1]).eql({headers: {Foo: "Bar"}});
+            expect(pullChanges.firstCall.args[1]).to.have.property("headers").eql({Foo: "Bar"});
```

Comment:
If the options have extra entries (defaut values or whatever unreleated to the test description here) they have to be mentionned. With this change we test only a subpart of the specified options, the ones concerned by the test (ie. headers here)